docs/cell_labels_in_testgrid.md

```diff
@@ -0,0 +1,94 @@
+In this document, our objective is to document the Interpretation of the cell values and cell labels in [testgrid](https://testgrid.k8s.io/):
+
+**Interpretation of cell values:**
+
+In this [file](https://github.com/GoogleCloudPlatform/testgrid/blob/a18fe953cf98174c215c43e0258b0515e37c283b/pb/test_status/test_status.proto#L3), we can see the meaning of different cell values.
+
+<table>
```

Comment:
Can you please make it a Markdown table?

https://www.markdownguide.org/extended-syntax/#tables

```markdown
| Value | Interpretation |
| ----- | -------------- |
| 0     | `NO_RESULT`    |
| 1     | `PASS`         |
...
```

Reply:
:+1: